modules/multi-arch-deleting-podplacmement-config-using-web-console.adoc

```diff
@@ -0,0 +1,41 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-deleting-podplacement-config-using-web-console_{context}"]
+
+= Deleting the ClusterPodPlacementConfig CR by using the web console
+
+You can delete the `ClusterPodPlacementConfig` custom resource (CR) by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+* You have created the `ClusterPodPlacementConfig` CR.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Click *Operators* → *Installed Operators*.
+
+. On the *Installed Operators* page, click *Multiarch Tuning Operator*. 
+
+. Click the *Pod Placement Config* tab.
+
+. Select *Delete ClusterPodPlacementConfig* from the options menu.
+
+. Click *Delete*.
+
+After deleting the `ClusterPodPlacementConfig` CR, you must delete the scheduling gate from all the pods by running the following command in the {oc-first}:
```

Comment:
This seems part of the section `Deleting the ClusterPodPlacementConfig CR by using the web console`, but it should be common to bot the web and cli version of removing the CR/Operand.

Also, I would highlight the concept that `some` pods can be left pending due to the scheduling gate not being handled anymore and that users `can` run this command to ensure that the scheduling gate is removed, if any pod is still pending due to that.

```
After deleting the `ClusterPodPlacementConfig` CR, you must ensure no pods are left pending due to the scheduling gate. You can run this command to unblock all the pods, if any:
```

Reply:
Updated as per the suggestion.